dbt/seeds/ccao/docs.md

```diff
@@ -8,6 +8,19 @@ regressions and reporting classes.
 **Primary Key**: `class_code`
 {% enddocs %}
 
+# loa
+
+{% docs seed_loa %}
+Table containing the Level of Assessment (LoA) for each minor property class
+for each year. LoAs change over time due to legislation, changes to Assessor
+class codes, etc.
+
+Do *not* use this table as a definitive list of existent class codes per year.
+Please use `ccao.class_dict` instead.
```

Comment:
**[Question, non-blocking]** `ccao.class_dict` doesn't yet have historical class codes, correct?

Reply:
Not yet, and it may not for awhile, but IMO it's fine for the moment.